### PR TITLE
Checkpoint3 Submission

### PR DESCRIPTION
**Pull request by lewisemm**

### CODE REVIEW

File: checkpoint3/urls.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	1. Add an import:  from other_app.views import Home
+	2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
+Including another URLconf
+	1. Import the include() function: from django.conf.urls import url, include
+	2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
+"""
+from django.conf.urls import url, include
+from django.contrib import admin
+
+from api import views
```
Comment (andela-anandaa) [inline checkpoint3/urls.py:19]: **Code Quality -- Nitpick**
This line is redundant.

**Comment by andela-anandaa** [comment]
#### README / Comments
- The README is detailed and okay.
- You could have a proper name for your app, other than Checkpoint3

#### Directory Structure
- Separate the API (DRF/Django app) and the Django app (front-end).

#### Git / Commit Messages
- Some of your git commit messages was not well wrapped, see how to fix this [here](http://stackoverflow.com/questions/11023194/git-vim-automatically-wrap-long-commit-messages)

#### Code Quality
- Generally okay except for a few _severe_ issues raised by [Quantified Code](https://www.quantifiedcode.com/app/project/gh:andela-lkabui:checkpoint3/diff/1591b1f8cf96be0c70c7624801f9234d23d53294/03af4f0b8f512e29de1d91bd47129909129fe550/issues?groups=code_patterns%3A4Phzg6k7%3Af1,code_patterns%3A5ZdRTEsh%3Af6,code_patterns%3Akv4o9Hx2,code_patterns%3A7I7lGdCx%3Af3&severity=4)

#### Functionality
- Works as expected
- Fix the loading of the first bucket-list
- Fix token expiry

#### Testing
- Good: 100% test coverage.

#### UI/UX
- Need a favicon
- Fix session expiry to be more usable
- Call-to-action button
- "Item Status" => "Done"
- The rest of the nitpicks [here](https://drive.google.com/a/andela.co/folderview?id=0B8IqKJACNaFOUzk3Y2hRNFlEbDA&usp=sharing)

**Comment by lewisemm** [comment]
Good Morning Nandaa,

I trust your morning is great.

Having had the checkpoint3 review last week, I have addressed the nitpicks we identified during our session and I am ready to present the improvements.

I am hoping we can have some time before the end of this week (today if possible) to review the changes.

Please tell me if this works for you.

> On 18 Feb 2016, at 11:34, Anthony Nandaa notifications@github.com wrote:
> 
> README / Comments
> 
> The README is detailed and okay.
> You could have a proper name for your app, other than Checkpoint3
> Directory Structure
> 
> Separate the API (DRF/Django app) and the Django app (front-end).
> Git / Commit Messages
> 
> Some of your git commit messages was not well wrapped, see how to fix this here http://stackoverflow.com/questions/11023194/git-vim-automatically-wrap-long-commit-messages
> Code Quality
> 
> Generally okay except for a few severe issues raised by Quantified Code https://www.quantifiedcode.com/app/project/gh:andela-lkabui:checkpoint3/diff/1591b1f8cf96be0c70c7624801f9234d23d53294/03af4f0b8f512e29de1d91bd47129909129fe550/issues?groups=code_patterns%3A4Phzg6k7%3Af1,code_patterns%3A5ZdRTEsh%3Af6,code_patterns%3Akv4o9Hx2,code_patterns%3A7I7lGdCx%3Af3&severity=4
> Functionality
> 
> Works as expected
> Fix the loading of the first bucket-list
> Fix token expiry
> Testing
> 
> Good: 100% test coverage.
> UI/UX
> 
> Need a favicon
> Fix session expiry to be more usable
> Call-to-action button
> "Item Status" => "Done"
> The rest of the nitpicks here https://drive.google.com/a/andela.co/folderview?id=0B8IqKJACNaFOUzk3Y2hRNFlEbDA&usp=sharing
> —
> Reply to this email directly or view it on GitHub https://github.com/andela-lkabui/checkpoint3/pull/5#issuecomment-185597691.

Lewis Kabui / Andela.com http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature 
Software Developer / +254726348287 / @Andela http://bit.ly/andelatwitter

"The startup that's harder to get into than Harvard" - CNN http://snip.ly/HfId
See what The Wall Street Journal http://snip.ly/u9AI has to say.

**Comment by lewisemm** [comment]
I’ve just realised I did not send the link. Please find my repo here https://github.com/andela-lkabui/checkpoint3/pull/12.

> On 25 Feb 2016, at 11:34, Lewis Maina lewis.maina@andela.com wrote:
> 
> Good Morning Nandaa,
> 
> I trust your morning is great.
> 
> Having had the checkpoint3 review last week, I have addressed the nitpicks we identified during our session and I am ready to present the improvements.
> 
> I am hoping we can have some time before the end of this week (today if possible) to review the changes.
> 
> Please tell me if this works for you.
> 
> > On 18 Feb 2016, at 11:34, Anthony Nandaa <notifications@github.com <mailto:notifications@github.com>> wrote:
> > 
> > README / Comments
> > 
> > The README is detailed and okay.
> > You could have a proper name for your app, other than Checkpoint3
> > Directory Structure
> > 
> > Separate the API (DRF/Django app) and the Django app (front-end).
> > Git / Commit Messages
> > 
> > Some of your git commit messages was not well wrapped, see how to fix this here http://stackoverflow.com/questions/11023194/git-vim-automatically-wrap-long-commit-messages
> > Code Quality
> > 
> > Generally okay except for a few severe issues raised by Quantified Code https://www.quantifiedcode.com/app/project/gh:andela-lkabui:checkpoint3/diff/1591b1f8cf96be0c70c7624801f9234d23d53294/03af4f0b8f512e29de1d91bd47129909129fe550/issues?groups=code_patterns%3A4Phzg6k7%3Af1,code_patterns%3A5ZdRTEsh%3Af6,code_patterns%3Akv4o9Hx2,code_patterns%3A7I7lGdCx%3Af3&severity=4
> > Functionality
> > 
> > Works as expected
> > Fix the loading of the first bucket-list
> > Fix token expiry
> > Testing
> > 
> > Good: 100% test coverage.
> > UI/UX
> > 
> > Need a favicon
> > Fix session expiry to be more usable
> > Call-to-action button
> > "Item Status" => "Done"
> > The rest of the nitpicks here https://drive.google.com/a/andela.co/folderview?id=0B8IqKJACNaFOUzk3Y2hRNFlEbDA&usp=sharing
> > —
> > Reply to this email directly or view it on GitHub https://github.com/andela-lkabui/checkpoint3/pull/5#issuecomment-185597691.
> 
> Lewis Kabui / Andela.com http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature 
> Software Developer / +254726348287 / @Andela http://bit.ly/andelatwitter
> 
> "The startup that's harder to get into than Harvard" - CNN http://snip.ly/HfId
> See what The Wall Street Journal http://snip.ly/u9AI has to say.

Lewis Kabui / Andela.com http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature 
Software Developer / +254726348287 / @Andela http://bit.ly/andelatwitter

"The startup that's harder to get into than Harvard" - CNN http://snip.ly/HfId
See what The Wall Street Journal http://snip.ly/u9AI has to say.

**Comment by lewisemm** [comment]
Hi Nandaa,

I hope you are well today.

This is just a follow up on my request to submit checkpoint3 after the changes from the review.

Please advise on what time would work best for you.

Regards.

> On 25 Feb 2016, at 12:08, Lewis Maina lewis.maina@andela.com wrote:
> 
> I’ve just realised I did not send the link. Please find my repo here https://github.com/andela-lkabui/checkpoint3/pull/12.
> 
> > On 25 Feb 2016, at 11:34, Lewis Maina <lewis.maina@andela.com <mailto:lewis.maina@andela.com>> wrote:
> > 
> > Good Morning Nandaa,
> > 
> > I trust your morning is great.
> > 
> > Having had the checkpoint3 review last week, I have addressed the nitpicks we identified during our session and I am ready to present the improvements.
> > 
> > I am hoping we can have some time before the end of this week (today if possible) to review the changes.
> > 
> > Please tell me if this works for you.
> > 
> > > On 18 Feb 2016, at 11:34, Anthony Nandaa <notifications@github.com <mailto:notifications@github.com>> wrote:
> > > 
> > > README / Comments
> > > 
> > > The README is detailed and okay.
> > > You could have a proper name for your app, other than Checkpoint3
> > > Directory Structure
> > > 
> > > Separate the API (DRF/Django app) and the Django app (front-end).
> > > Git / Commit Messages
> > > 
> > > Some of your git commit messages was not well wrapped, see how to fix this here http://stackoverflow.com/questions/11023194/git-vim-automatically-wrap-long-commit-messages
> > > Code Quality
> > > 
> > > Generally okay except for a few severe issues raised by Quantified Code https://www.quantifiedcode.com/app/project/gh:andela-lkabui:checkpoint3/diff/1591b1f8cf96be0c70c7624801f9234d23d53294/03af4f0b8f512e29de1d91bd47129909129fe550/issues?groups=code_patterns%3A4Phzg6k7%3Af1,code_patterns%3A5ZdRTEsh%3Af6,code_patterns%3Akv4o9Hx2,code_patterns%3A7I7lGdCx%3Af3&severity=4
> > > Functionality
> > > 
> > > Works as expected
> > > Fix the loading of the first bucket-list
> > > Fix token expiry
> > > Testing
> > > 
> > > Good: 100% test coverage.
> > > UI/UX
> > > 
> > > Need a favicon
> > > Fix session expiry to be more usable
> > > Call-to-action button
> > > "Item Status" => "Done"
> > > The rest of the nitpicks here https://drive.google.com/a/andela.co/folderview?id=0B8IqKJACNaFOUzk3Y2hRNFlEbDA&usp=sharing
> > > —
> > > Reply to this email directly or view it on GitHub https://github.com/andela-lkabui/checkpoint3/pull/5#issuecomment-185597691.
> > 
> > Lewis Kabui / Andela.com http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature 
> > Software Developer / +254726348287 / @Andela http://bit.ly/andelatwitter
> > 
> > "The startup that's harder to get into than Harvard" - CNN http://snip.ly/HfId
> > See what The Wall Street Journal http://snip.ly/u9AI has to say.
> 
> Lewis Kabui / Andela.com http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature 
> Software Developer / +254726348287 / @Andela http://bit.ly/andelatwitter
> 
> "The startup that's harder to get into than Harvard" - CNN http://snip.ly/HfId
> See what The Wall Street Journal http://snip.ly/u9AI has to say.

Lewis Kabui / Andela.com http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature 
Software Developer / +254726348287 / @Andela http://bit.ly/andelatwitter

"The startup that's harder to get into than Harvard" - CNN http://snip.ly/HfId
See what The Wall Street Journal http://snip.ly/u9AI has to say.

**Pull request merged**